image/cli-base/install/install-oc.sh

```diff
@@ -41,6 +41,6 @@ chmod +x /usr/local/bin/oc-mirror
 rm -f oc-mirror.tar.gz
 
 echo "oc-mirror version:"
-oc-mirror version --output json
+#oc-mirror version --output json
```

Comment:
Why do we remove this debug line? 

If we are removing this line for valid reasons, we should probably remove the echo line above it too?

Reply:
removed the echo line 
Gives  below error in arm64 and in s390x .
#12 15.48 + oc-mirror version --output json
#12 15.55 FATAL: ThreadSanitizer: unsupported VMA range